products/jquery-ui.md

```diff
@@ -0,0 +1,97 @@
+---
+title: jQuery UI
+category: framework
+tags: javascript-runtime
+permalink: /jquery-ui
+changelogTemplate: https://github.com/jquery/jquery-ui/releases/tag/__LATEST__
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:github/jquery/jquery-ui
+-   purl: pkg:maven/org.webjars/jquery-ui
+-   purl: pkg:npm/jquery-ui
+-   purl: pkg:nuget/jQuery.UI.Combined
+
+auto:
+  methods:
+  -   git: https://github.com/jquery/jquery-ui.git
+
+releases:
+-   releaseCycle: "1.14"
+    releaseDate: 2024-08-05
+    eol: false
+    latest: "1.14.0"
+    link: https://blog.jqueryui.com/2024/08/jquery-ui-1-14-0-released/
+
+-   releaseCycle: "1.13"
+    releaseDate: 2021-10-07
+    eol: true
+    latest: "1.13.3"
+    latestReleaseDate: 2024-04-26
+    link: https://blog.jqueryui.com/2024/04/jquery-ui-1-13-3-released/
+
+-   releaseCycle: "1.12"
+    releaseDate: 2016-04-21
+    eol: true
+    latest: "1.12.1"
+    latestReleaseDate: 2016-09-15
+    link: https://blog.jqueryui.com/2016/09/jquery-ui-1-12-1/
+
+-   releaseCycle: "1.11"
+    releaseDate: 2014-06-26
+    eol: true
+    latest: "1.11.4"
+    latestReleaseDate: 2015-03-11
+    link: https://blog.jqueryui.com/2015/03/jquery-ui-1-11-4/
+
+-   releaseCycle: "1.10"
+    releaseDate: 2013-01-17
+    eol: true
+    latest: "1.10.4"
+    latestReleaseDate: 2014-01-17
+    link: https://blog.jqueryui.com/2014/01/jquery-ui-1-10-4/
+
+-   releaseCycle: "1.9"
+    releaseDate: 2012-10-08
+    eol: true
+    latest: "1.9.2"
+    latestReleaseDate: 2012-11-23
+    link: https://blog.jqueryui.com/2012/11/jquery-ui-1-9-2/
+
+-   releaseCycle: "1.8"
+    releaseDate: 2010-03-23
+    eol: true
+    latest: "1.8.24"
+    latestReleaseDate: 2012-09-28
+    link: https://blog.jqueryui.com/2012/09/jquery-ui-1-8-24/
+
+-   releaseCycle: "1.7"
+    releaseDate: 2009-03-06
+    eol: true
+    latest: "1.7.3"
+    latestReleaseDate: 2010-05-04
+    link: https://blog.jqueryui.com/2010/05/jquery-ui-173/
+
+---
+
+
+> [jQuery UI](https://jqueryui.com/) is a collection of GUI widgets, animated
+> visual effects, and themes implemented with jQuery.
+
+
+Only the latest version line is supported. The 1.13.x and below no longer receive
+patches.
+
+## [Browser Support](https://jqueryui.com/browser-support/)
+
+jQuery UI 1.14.x supports the current stable version of each supported browser.  
+jQuery UI 1.13.x was the last version line to support IE 11 and Edge Legacy 18.
+
+| Browser | Support Versions      |
+|---------|-----------------------|
+| Chrome  | Latest stable release |
+| Edge    | Latest stable release |
+| Firefox | Latest stable release |
+| Safari  | Latest stable release |
+
```

Comment:
No point in replicating the table, or mentioning 1.14.x (since only
the latest version is supported, that is implied). Mentioning
1.13 (an unsupported release) is also not helpful, since there
be many other browsers that are not supported.

```suggestion
JQuery UI supports the latest stable releases of Chrome, Edge, Firefox,
and Safari. While it might run without major issues in older browser
versions, it is not actively tested against and bug fixes only impacting
other browsers might not be fixed.
```

In general, we avoid using specific releases in description, since it requires a lot more maintenance. We can easily get away without that here.